acme/commons.go

```diff
@@ -46,6 +46,7 @@ type Directory struct {
 	RevokeCertURL string `json:"revokeCert"`
 	KeyChangeURL  string `json:"keyChange"`
 	Meta          Meta   `json:"meta"`
+	RenewalInfo   string `json:"renewalInfo"`
```

Comment:
Add this link so that people know where the RenewalInfo object originated.
https://www.ietf.org/archive/id/draft-ietf-acme-ari-01.html#name-extensions-to-the-acme-prot

or maybe
`// - https://datatracker.ietf.org/doc/draft-ietf-acme-ari/ (3. Extensions to the ACME Protocol: The "directory" Resource)`